### PR TITLE
Support for bitwise operations

### PR DESCRIPTION
**Pull request by ashutoshparkhi**
Support for bitwise operators in tensorflow frontend and NNVM top

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

**Comment by ashutoshparkhi** [comment]
@yzhliu could you please add reviewers?

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 1]
```diff
@@ -1047,6 +1047,49 @@ def test_forward_rel_ops():
     _test_forward_rel_op([t1, t2], math_ops.equal)
     _test_forward_rel_op([t1, t2], math_ops.not_equal)
 
+#######################################################################
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Suggest to add TOPI level test cases with all targets enabled.

Comment (ashutoshparkhi) [reply nnvm/tests/python/frontend/tensorflow/test_forward.py]: @srkreddy1238 added new tests. Please take a look.

**Comment by icemelon9** [comment]
@Huyuwei @Laurawly Please help review this PR

**Pull request closed without merging**